Lambdas/GHHooks/Extensions/+Issue.Label.swift

```diff
@@ -0,0 +1,29 @@
+import GitHubAPI
+
+extension Issue {
+    enum KnownLabel: String {
+        case enhancement
+        case bug
```

Comment:
Might as well recognize `duplicate`, `wontfix`, and `invalid` here while we're at it.